content/src/main/content/jcr_root/apps/acs-commons/components/utilities/errorpagehandler/404.jsp

```diff
@@ -26,12 +29,23 @@
         // Check for and handle 404 Requests properly according on Author/Publish
         if (errorPageHandlerService.doHandle404(slingRequest, slingResponse)) {
 
-            final String path = errorPageHandlerService.findErrorPage(slingRequest, resource);
+            SlingHttpServletRequest req = sling.getRequest();
 
+            String resourcePath = StringUtils.removeEnd(resource.getPath(),resource.getName());
+
+            Resource res = resource.getResourceResolver().resolve(req , resourcePath);
+
+
```

Comment:
It might make more sense to try to always get the aggregator node, since once this resource if found, it tried to get a "jcr:content" child

```
StringUtils.substringBefore(resource.getPath(), "jcr:content");
```

So...

Given a resource with path ~> the resource to get the jcr:content node for
````
/content/site/en.html ~> /content/site/en
/content/site/en/jcr:content ~> /content/site/en
/content/site/en/jcr:content/title ~> /content/site/en
/content/site/en/jcr:content/par/subtitle ~> /content/site/en
```

I believe this should work for DAM too (slingOrderedFolders)

````
/content/dam/site/docs~> /content/dam/site/docs
/content/dam/site/docs/cat.png ~> /content/dam/site/docs/cat.png
/content/dam/site/docs/cat.png/jcr:content/renditions/some-rendition.png ~> /content/dam/site/docs/cat.png
```